pkg/stash/with_azureblob_test.go

```diff
@@ -82,14 +84,28 @@ func (ms *mockAzureBlobStasher) Stash(ctx context.Context, mod, ver string) (str
 	return "", fmt.Errorf("second time error")
 }
 
-func getAzureTestConfig() *config.AzureBlobConfig {
+func getAzureTestConfig(containerName string) *config.AzureBlobConfig {
 	key := os.Getenv("ATHENS_AZURE_ACCOUNT_KEY")
 	if key == "" {
 		return nil
 	}
+	name := os.Getenv("ATHENS_AZURE_ACCOUNT_NAME")
+	if name == "" {
+		return nil
+	}
 	return &config.AzureBlobConfig{
-		AccountName:   "athens_drone_azure_account",
+		AccountName:   name,
 		AccountKey:    key,
-		ContainerName: "athens_drone_azure_container",
+		ContainerName: containerName,
+	}
+}
+
+func randomContainerName(prefix string) string {
+	// moniker is a cool library to produce mostly unique, human-readable names
+	// see https://github.com/technosophos/moniker for more details
+	namer := moniker.New()
+	if prefix != "" {
+		return fmt.Sprintf("%s_%s", prefix, namer.NameSep(""))
```

Comment:
@marpio looks like container names can't have underscores. maybe change this to a dash?  here's some sample code for how to create container names: https://github.com/Azure-Samples/storage-blobs-go-quickstart/blob/master/storage-quickstart.go#L30

Docs for container names: https://docs.microsoft.com/en-us/rest/api/storagerp/blobcontainers/create